bdsf/multi_proc.py

```diff
@@ -203,6 +203,7 @@ def parallel_map(function, sequence, numcores=None, bar=None, weights=None):
     if weights is None or numcores == size:
         # No grouping specified (or there are as many cores as
         # processes), so divide into equal chunks
+        sequence = numpy.array(sequence, dtype=object)
```

Comment:
Your fix only applies to the `if` part, not the `else` part.
I propose to make the fix in line 161 instead by changing that to:
```
sequence = numpy.array(list(sequence), dtype=object)
```

Reply:
Good catch. I will use your suggested fix.